src/main/scala/dsptools/DspTester.scala

```diff
@@ -247,12 +247,11 @@ class DspTester[T <: Module](c: T,
     }
   }
 
-  def dspExpect(data: DspComplex[_], expected: Complex, msg: String): Unit = {
+  def dspExpect(data: DspComplex[_], expected: Complex, msg: String): Boolean = {
     dspPeek(data) match {
       case Right(complex) =>
         println(f"expect got $complex expect $expected")
-        expect(nearlyEqual(complex.real, expected.real), msg)
-        expect(nearlyEqual(complex.imag, expected.imag), msg)
+        expect(nearlyEqual(complex.real, expected.real), msg) & expect(nearlyEqual(complex.imag, expected.imag), msg)
```

Comment:
lol I guess I use & and && interchangeably in a bad way... since it works with Booleans...